NotebookAnalyzers/NotebooksAnalysisResults.ipynb

```diff
@@ -53,8 +55,9 @@
     "data=[]\n",
     "for filename in os.listdir(config.data_path):\n",
     "       if filename.endswith(\".ipynb\"):\n",
-    "              script = pynblint.notebook_to_script(filename)\n",
-    "              nb_dict = pynblint.notebook_to_dict(filename)\n",
+    "              notebook = entities.Notebook(filename)\n",
+    "              script = notebook.get_code()\n",
+    "              nb_dict = notebook.get_dict()\n",
```

Comment:
I'd rather pass whole `Notebook` objects to the functions of the `pynblint` library.

It should be a responsibility of each function to take the notebook representation it needs from input `Notebook` objects.